pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherSingleActiveConsumer.java

```diff
@@ -68,6 +68,7 @@
     private volatile ScheduledFuture<?> readOnActiveConsumerTask = null;
 
     private final RedeliveryTracker redeliveryTracker;
+    private long startTimeNs;
```

Comment:
Why use `Ns` variable name for millseconds? it might be confusing

Reply:
OK,I will fix.

Reply:
I have fixed：use  nanoTime for startTimeNs;
please review again!